doofinder-for-woocommerce/readme.txt

```diff
@@ -1,9 +1,9 @@
 === Doofinder for WooCommerce ===
-Contributors: doofinder, chopchoporg
-Tags: search, autocomplete, woo commerce, e-commerce, ecommerce, sales, shop
-Version: 1.2.22
-Requires at least: 5.0
-Tested up to: 5.4.1
```

Comment:
Why this?

Reply:
@carlosescri Again, i didn't mean to remove this. Not sure how that happened.